packages/connect-popup/src/index.tsx

```diff
@@ -59,12 +60,19 @@ const handleMessage = (
     }
 
     if (data.type === RESPONSE_EVENT && !data.success) {
```

Comment:
I believe something like 
```
if (data.type === RESPONSE_EVENT && !data.success) && data.payload?.code !== 'Transport_Missing')
```
should be enough here. no dispatching should be needed, as it happens on lines 115, 121